grudge/dof_desc.py

```diff
@@ -154,11 +170,12 @@ def __init__(self, domain_tag, quadrature_tag=None):
             :class:`~meshmode.mesh.BTAG_PARTITION`,
             or *None* to indicate that the geometry is not yet known.
 
-        :arg quadrature_tag:
-            *None* to indicate that the quadrature grid is not known, or
-            :class:`QTAG_NONE` to indicate the use of the basic discretization
-            grid, or a string to indicate the use of the thus-tagged quadratue
-            grid.
+        :arg discretization_tag:
+            *None* to indicate that the discretization grid is not known,
+            :class:`DISCR_TAG_BASE` to indicate the use of the basic
+            discretization grid, :class:`DISCR_TAG_MODAL` to indicate a
+            modal discretization, or :class:`DISCR_TAG_QUAD` or a string
+            to indicate the use of the thus-tagged quadrature grid.
```

Comment:
I'd say we deprecate (and don't document) string discretization tags.

Reply:
Done: https://github.com/inducer/grudge/pull/83/commits/a70b9d6f0b904382bed836331f6bb043cb6ea297